src/zope/publisher/tests/test_browserresponse.py

```diff
@@ -167,30 +167,28 @@ def test_handleException(self):
         response = BrowserResponse()
         try:
             raise ValueError(1)
-        except:
+        except:  # noqa: E722 do not use bare 'except'
             exc_info = sys.exc_info()
 
         response.handleException(exc_info)
         self.assertEqual(response.getHeader("content-type"),
-            "text/html;charset=utf-8")
+                         "text/html;charset=utf-8")
         self.assertEqual(response.getStatus(), 500)
-        self.assertTrue(response.consumeBody() in
-            [b"<html><head><title>&lt;type 'exceptions.ValueError'&gt;</title></head>\n"
+        self.assertTrue(
+            response.consumeBody() in
```

Comment:
How did you resist the temptation to change this to
```suggestion
        self.assertIn(
            response.consumeBody(),
```

Reply:
Fixed this one and some others.